### DISCUSSION TITLE
[WASM] Warning in VS after updating to UNO 4.8

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#11766] [General]
I see this warning in my VS output after updating to UNO 4.8. 
`Microsoft.AspNetCore.Watch.BrowserRefresh.BrowserRefreshMiddleware: Warning: Unable to configure browser refresh script injection on the response. Consider manually adding '<script src="/_framework/aspnetcore-browser-refresh.js"></script>' to the body of the page.
`

What's the proper way to resolve this?

**Comment by jeromelaban** [comment]
Thanks for the report. This is unlikely to be caused by Uno.WinUI 4.8. Did you also upgrade any `uno.wasm.*` packages, and does it happen again if you downgrade?

**Reply by DierkDroth** [reply]
Thanks @jeromelaban. You're right: I even see those warnings when downgrading to the prior versions as per screenshot below.

What would cause these warnings and how could I resolve them?

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/26179242/226897524-3fab86bb-8eb7-419c-bbd4-6eb622385226.png">

**Reply by jeromelaban** [reply]
It may be a Visual Studio recent issue. Did you upgrade it recently?

**Reply by DierkDroth** [reply]
Yes, I upgraded it yesterday or the day before to the latest official version (pls see screenshot)

<img width="576" alt="image" src="https://user-images.githubusercontent.com/26179242/226898560-ee55dbc4-a146-4194-9b33-6407e04146d0.png">

**Reply by jeromelaban** [reply]
Thanks. I wonder if https://github.com/unoplatform/Uno.Wasm.Bootstrap/pull/604 may be a problem. Can you try reverting back to `uno.wasm.bootstrap*` to 7.0.5 (if possible) and see if it helps?

**Reply by DierkDroth** [reply]
Downgrading (screenshot) unfortunately still yields the same warnings on startup

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/26179242/226903325-a712c0b6-91e2-4837-b74d-069d99c09ecd.png">

**Reply by jeromelaban** [reply]
Have you downgraded all bootstrapper packages? The DevServer one needs to be as well.

**Reply by DierkDroth** [reply]
Unfortuantely downgrading both packages (screenshot) does not resolve the issue.

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/26179242/226905753-fb3fae6f-703d-4d04-878f-914569e6e9e3.png">

**Reply by DierkDroth** [reply]
@jeromelaban would you have any more hints on what could be done to resolve the issue?

**Comment by DierkDroth** [comment]
FYI for anyone running into that issue.

Appears this is related [to hot reload ](https://learn.microsoft.com/en-us/answers/questions/1007447/what-is-aspnetcore-browser-refresh-js-and-how-can)

Since I don't need hot reload, I deactivated it [as per here](https://learn.microsoft.com/en-us/aspnet/core/test/hot-reload?view=aspnetcore-6.0) by adding `"hotReloadEnabled" : false` to my `Properties/launchSettings.json`
